FHIR_README.md

```diff
@@ -44,6 +44,7 @@ Database Result -> Service Component -> FHIR Service Component -> Parse OpenEMR
 -   [FHIR API Endpoints](FHIR_README.md#fhir-endpoints)
     -   [Capability Statement](FHIR_README.md#capability-statement)
     -   [Patient](FHIR_README.md#patient-resource)
+    -   [Coverage](FHIR_README.md#coverage-resource)
```

Comment:
Also need to add this to the API_README.md table of contents (for some reason we also have links to the FHIR endpoints there also).

Reply:
Done. Little surprised that I missed it in the first run :)